### ISSUE TITLE
Add RSS button so that people know they can subscribe to our feed.

### ISSUE THREAD

**Issue by austinkocher** [LawandSpace/Law-Space-Website#3]

**Comment by Riwajchalise** [comment]
Can you please elaborate on this? do you mean the News letter?

**Comment by austinkocher** [comment]
Yes, certainly. The website creates an RSS feed that I already subscribed to just to test it. However, there's no specific button/link on the website where people can click to add it to their RSS reader. I don't know if this is a good idea or not. It was really just a thought.

**Comment by Riwajchalise** [comment]
Sounds like a good idea. How about using mail chim in our blog It already comes with the theme. I think we need to enable and configure it to collect email list of our subsccribers.

**Comment by austinkocher** [comment]
That's a good idea. I do want to create an email option in the near future, but I'm not ready for that yet. We can put this on the back burner for now.

**Comment by Riwajchalise** [comment]
Yes this would be a good idea once the site starts to get some traffic.
